irma-core/state-machine.js

```diff
@@ -99,20 +97,19 @@ module.exports = class StateMachine {
       this._disabledTransitions = payload.canRestart ? [] : ['restart'];
 
     this._listeners.forEach(func => func({
-      newState: this._state,
-      oldState: oldState,
+      newState:   this._state,
+      oldState:   oldState,
```

Comment:
This is actually funny, if you compare it with `master` instead of with `shoulder-surf` there is no diff here. So on the `shoulder-surf` brach I apparently changed this. It was not really necessary to change this, so I reverted it here again. Let's wait with reformatting until we have a decent formatter.

Reply:
Ah makes sense